src/app_main.c

```diff
@@ -55,7 +56,14 @@ void app_main() {
 
     io_init();
 
-    ui_menu_main();
+#ifdef HAVE_SWAP
```

Comment:
Not sure we need to handle the `!HAVE_SWAP` case
I say we assume SWAP is always enabled and that's it, we do so in all other applications